docs/projects/index.md

```diff
@@ -4,11 +4,24 @@ The regular disclosure of structured data can greatly enhance the transparency a
 
 ## What is a project?
 
-An infrastructure project may stand alone (e.g. a town building a new hospital), or may form part of a wider programme of activity (e.g. construction of a number of railway stations). Each project will usually involve project-level planning and co-ordination, managed through a series of contracts and subcontracts.
+An infrastructure project may stand alone (e.g. a town building a new hospital), or may form part of a wider programme of activity (e.g. construction of a number of railway stations). Each infrastructure project will usually involve project-level planning and co-ordination, managed through a series of contracts and subcontracts. 
 
-![Projects typically involve multiple contracts](../../_static/images/diagram-project-contract.png)
+```eval_rst
+.. admonition:: Tip
+    :class: tip
+
+    .. markdown::
+
+      The term 'project' can also be used to refer to a larger programme of work of which an individual infrastructure project is a part, such as the construction of a whole highway. The term 'project' may also refer to a public investment project with funds allocated in the national budget, such as the upgrading of the rail network in a particular region.
```

Comment:
I don't remember this being clarified anywhere else in the documentation. We should at minimum update the project schema to define the project in this way. Some updated text:

`In the context of OC4IDS, the term 'project' refers to the construction of a **single asset**: for example, the construction of a bridge, highway junction or building.`

`In other contexts, the term 'project' can refer to a larger programme of work – of which projects in the sense of OC4IDS would form part of – like the construction of an entire highway. The term 'project' can also refer to a public investment, to which funds are allocated in the national budget, like the electrification of a rail network.`

Reply:
Fixed in https://github.com/open-contracting/infrastructure/pull/106/commits/cee03c44e6d293a00091289a07eaeb852e6af615 and project definition updated in https://github.com/open-contracting/infrastructure/pull/106/commits/a625e5bf13f16918b3a07dd2069f6689aa170cf8

Reply:
I checked back to [guidance note 6](http://infrastructuretransparency.org/wp-content/uploads/2011/12/31_CoST_Guidance_Note_6_Proof.pdf) and the use of 'single' asset was over specific, so I've updated the documentation and schema to use the definition of 'project' from the guidance note:

> the delivery of a set of infrastructure assets in a specified location, generally under a single managing entity (project owner) and budget authority